### ISSUE TITLE
set ENV["DATADEPS_ALWAYS_ACCEPT"] = true

### ISSUE THREAD

**Issue by CarloLucibello** [JuliaML/MLDatasets.jl#103]
Having to say "yes" each time seems just an annoyance.

I think this should be considered a breaking change.

**Comment by johnnychen94** [comment]
I disagree. We can't represent our users to agree on the terms. He should agree on the terms by himself, or set the environment by himself.

**Comment by johnnychen94** [comment]
A possible alternative to make it smoother is to really highlight the license note somewhere so that everyone knows it.

For instance, it can be put to https://juliaml.github.io/MLDatasets.jl/stable/#Installation with something like

```
!!! note "License"
    This package only provides a convenient API to read commonly used datasets from the internet. Make
    sure you follow the license requirement of the dataset you're trying to use.
```

**Comment by alecloudenback** [comment]
I'm trying to run some code on a remote machine and it's getting stuck at the step of saying "yes" to downloading the data. How can I say yes programmatically rather than via the REPL?

**Comment by CarloLucibello** [comment]
Just put
```julia
ENV["DATADEPS_ALWAYS_ACCEPT"] = true
```
in your julia code.
